test/JDBC/expected/Test-spatial-functions-vu-verify.out

```diff
@@ -1191,17 +1191,6 @@ int#!#float#!#text
 ~~END~~
 
 
-DECLARE @point geometry = geometry::Point(1.0, 2.0, 4326);
-SELECT @point.STSrid AS SRID, COUNT(*) AS PointCount 
-FROM TestSpatialFunction_YourTableTemp GROUP BY PointColumn.STSrid;
-GO
-~~START~~
-int#!#int
-4326#!#2
-4326#!#1
-~~END~~
```

Comment:
why we have to remove this test case

Reply:
The output in local JDBC tests
int#!#int
4326#!#2
4326#!#1

the output in upgrade tests
int#!#int
4326#!#1
4326#!#2

but resolved as of now Added order by clause